packages/gatsby-theme-carbon/src/components/Header/Header.module.scss

```diff
@@ -56,7 +56,6 @@ header.header {
   transition: opacity $duration--fast-02 $carbon--standard-easing;
   color: $inverse-01;
   text-decoration: none;
-  position: absolute;
```

Comment:
Hmmm I wonder if this was here for a reason o.o

Reply:
Hmm I don't remember deleting this - I must have committed it on accident - I shouldn't have / didn't want to change anything to this module. Adding it back.